planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -634,6 +636,11 @@ void planning_scene_monitor::PlanningSceneMonitor::excludeRobotLinksFromOctree()
       if (h)
         link_shape_handles_[links[i]].push_back(std::make_pair(h, j));
     }
+    if(!warned && ((ros::Time::now() - start) > ros::Duration(30.0)))
+    {
+      ROS_WARN_STREAM_NAMED("PlanningSceneMonitor::excludeRobotLinksFromOctree()", "It is likely there are too many vertices in collision geometry");
+      warned = true;
```

Comment:
Thank you for looking into the performance, good point.

Still, "This warning to me belongs at the location where the meshes are first loaded into the planning scene." Can we track down where that is? Probably geometric_shapes is involved